src/components/Transactions.tsx

```diff
@@ -25,7 +25,7 @@ import { useTransactionContext } from '../contexts/TransactionContext';
 import { useUpdateTransactionContext } from '../contexts/TransactionContext';
 import { TransactionStatusEnum, TransanctionStatus } from '../types/transactionTypes';
 import shortenItem from '../util/shortenItem';
-import TransactionStatus from './TransactionStatus';
+import { TransactionStatus } from '.';
```

Comment:
I feel the `index.ts` files are unnecessary and are just boilerplate to maintain. I'm also quite reluctant to believe that introducing an extra "hub" file helps with tree shaking in any way.

Reply:
The actual case is that it makes the export of components to be done straight from the components dir. so instead of:
`import Account from './components/Account';`
it helps to change to:
`import { Account } from './components';`
The example above may look not-much-diff but in case of multiple imports, that reduces the lines and readability.

If though this is not a preferable way I can just rollback - just let me know.